pkg/telemetry/telemetry.go

```diff
@@ -140,14 +141,37 @@ func (c Component) sendTelemetry() {
 		c.log.WithError(err).Warning("can't prepare telemetry data")
 		return
 	}
-	c.log.WithField("data", data).Info("sending telemetry")
+
+	hostData := analytics.Context{
+		Direct: true,
+	}
+
+	hostData.App.Version = c.Version
+	hostData.App.Name = "k0s"
+	hostData.App.Namespace = "k0s"
+	hostData.Extra["cpuArch"] = runtime.GOARCH
+
+	if runtime.GOOS == "linux" {
+		var si sysinfo.SysInfo
+		si.GetSysInfo()
```

Comment:
Not sure if this gets heavy, it does all kinds of data retrieval from who knows where.

Reply:
the telemetry is collected anyway once per 10mins by default, so IMO not an issue